source/query/filter.txt

```diff
@@ -47,6 +47,283 @@ Set Query Filter
+
+   If you do not have a MongoDB deployment or if you want to query a
+   larger sample data set, see :atlas:`Sample Data for Atlas Clusters
+   </sample-data/>` for instructions on creating a free-tier cluster
+   with sample data. The following example queries filter the sample
+   documents provided on this page.
+
+Match by a Single Condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+         
+The following query filter finds all documents where the value of
+``name`` is "Andrea Le":
+
+.. code-block:: shell
+
+   { name: "Andrea Le" }
+
+The query returns the following document:
+
+.. code-block:: JSON
+   :copyable: false
+
+   {
+      "_id": { "$oid": "5e349915cebae490877d561d" },
+      "name": "Andrea Le",
+      "email": "andrea_le@fake-mail.com",
+      "version": 5,
+      "scores": [ 85, 95, 75 ],
+      "dateCreated": { "$date": "2003-03-26" }
+   }
+
+Match by Multiple Conditions ($and)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following query filter finds all documents where ``scores`` array
+contains the value ``75``, and the ``name`` is ``Greg Powell``:
+
+.. code-block:: shell
+
+   { scores: 75, name: "Greg Powell" }
+
+The query returns the following document:
+
+.. code-block:: JSON
+   :copyable: false
+
+   {
+      "_id": { "$oid":"5a9427648b0beebeb69579cf" },
+      "name": "Greg Powell",
+      "email": "greg_powell@fake-mail.com",
+      "version": 1,
+      "scores": [ 65, 75, 80 ],
+      "dateCreated": { "$date": "1999-02-10" }
+   }
+
+Match by Multiple Possible Conditions ($or)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following query filter uses the :query:`$or` operator to find
+documents where ``version`` is ``4``, or ``name`` is ``Andrea Le``:
+
+.. code-block:: shell
+
+   { $or: [ { version: 4 }, { name: "Andrea Le" } ] }
+
+The query returns the following documents:
+
+.. code-block:: JSON
+   :copyable: false
+
+   [
+      {
+         "_id": { "$oid": "5e349915cebae490877d561d" },
+         "name": "Andrea Le",
+         "email": "andrea_le@fake-mail.com",
+         "version": 5,
+         "scores": [ 85, 95, 75 ],
+         "dateCreated": { "$date": "2003-03-26" }
+      },
+      {
+         "_id": { "$oid":"5e349915cebae490877d561e" },
+         "email": "no_name@fake-mail.com",
+         "version": 4,
+         "scores": [ 90, 90, 75 ],
+         "dateCreated": { "$date": "2001-04-15" }
+      }
+   ]
+
+Match by Exclusion ($not)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+         
+The following query filter uses the :query:`$not` operator to find all
+documents where the value of the ``name`` field is **not** equal to
+"Andrea Le", or the ``name`` field does not exist:
+
+.. code-block:: shell
+
+   { name: { $not: { $eq: "Andrea Le" } } }
+
+The query returns the following documents:
+
+.. code-block:: JSON
+   :copyable: false
+
+   [
+      {
+         "_id": { "$oid":"5e349915cebae490877d561e" },
+         "email": "no_name@fake-mail.com",
+         "version": 4,
+         "scores": [ 90, 90, 75 ],
```

Comment:
[typo]

The value for this field should be [ 90, 90, 70 ] according to the inserted documents.

Reply:
Fixed.